curriculum/dictionaries/spanish/comments.js

```diff
@@ -0,0 +1,454 @@
+  },
+  {
+    id: 'avpx79',
+    text: 'Abre la consola de tu navegador.'
+  },
+  {
+    id: '0b5ps6',
+    text: 'Relleno entre el límite del canvas SVG y el gráfico'
+  },
+  {
+    id: 'uemoej',
+    text: 'Inserta n ceros a la fila actual para crear las columnas'
+  },
+  {
+    id: 'lm86nf',
+    text: 'Inserta la fila actual, que ahora contiene n ceros, al arreglo'
+  },
+  {
+    id: 'qscelx',
+    text: 'Los métodos Redux están disponibles desde un objeto Redux'
+  },
+  {
+    id: 'atqiig',
+    text: 'Renderiza un input, button, y ul debajo de esta línea'
+  },
+  {
+    id: 'yq81wg',
+    text: 'Renderiza el Provider debajo de esta línea'
+  },
+  {
+    id: 'kxio9j',
+    text:
+      'responseFromServer es establecido a true para representar una respuesta no satisfactoria del servidor'
+  },
+  {
+    id: 'alh6pw',
+    text:
+      'responseFromServer es establecido a true para representar una respuesta satisfactoria del servidor'
+  },
+  {
+    id: '1cfidd',
+    text: 'responseFromServer representa una respuesta de un servidor'
+  },
+  {
+    id: '96tntk',
+    text: 'Retorna 30'
+  },
+  {
+    id: '58a5g7',
+    text: 'Ejecuta y verifica la consola'
+  },
+  {
+    id: '71bus9',
+    text: 'Ejecuta las pruebas para ver la diferencia entre las dos consolas.'
+  },
+  {
+    id: '7wp46n',
+    text: 'Ajuste en escala Farenheit'
+  },
+  {
+    id: 'oefvg5',
+    text: 'Configuración'
+  },
+  {
+    id: 'mnt4d3',
+    text: "Debería mostrar 'carrot'"
+  },
+  {
+    id: 'fhe9m4',
+    text: 'Sitios sociales'
+  },
+  {
+    id: 'za434b',
+    text: 'La tabla de barras apiladas de entrenamiento semanal'
+  },
+  {
+    id: '7c1fv9',
+    text: 'La tabla de barras apiladas irá aquí'
+  },
+  {
+    id: 'r44ovx',
+    text:
+      'tabs es un arreglo de títulos de cada sítio abierto dentro de la ventana'
+  },
+  {
+    id: 'cl8peb',
+    text: 'arreglo de prueba:'
+  },
+  {
+    id: '1xi3cv',
+    text: 'La variable global'
+  },
+  {
+    id: '3gc01a',
+    text: 'El archivo main.scss'
+  },
+  {
+    id: '14kfog',
+    text: 'Este es nuestro método de intersección'
+  },
+  {
+    id: 'd1shtt',
+    text: 'Este es nuestro método de unión'
+  },
+  {
+    id: 'pqq6sy',
+    text: 'Este método agregará un elemento al conjunto'
+  },
+  {
+    id: 'nd2oxy',
+    text:
+      'Este método verificará la existencia de un elemento y retornará true o false'
+  },
+  {
+    id: 'ocm81t',
+    text: 'Este método removerá un elemento del conjunto'
+  },
+  {
+    id: 'or9p5p',
+    text: 'Este método retornará todos los valores del conjunto'
+  },
+  {
+    id: 'g1608f',
+    text: 'Este método retornará el tamaño del conjunto'
+  },
+  {
+    id: 'bheu99',
+    text: 'Esto almacenará el conjunto'
+  },
+  {
+    id: 'x1djjr',
+    text:
+      'Usa console.clear() en la siguiente línea para limpiar la consola del navegador.'
+  },
+  {
+    id: '22ta95',
+    text: 'Usa console.log() para imprimir la variable de salida.'
```

Comment:
The English text for this id is:
> Use console.log() to print the output variable.

`output` is a variable name and should not be translated.